File: despatch.WEB/app/src/swagger/models/user-response.ts
/* tslint:disable */
import { IdentityUser } from './identity-user';
import { UserAddressResponse } from './user-address-response';
import { CityResponse } from './city-response';
import { CountryResponse } from './country-response';
import { UserImageResponse } from './user-image-response';
import { UserOrganizationResponse } from './user-organization-response';
import { UserPhoneNumbersResponse } from './user-phone-numbers-response';
import { UserUrlsResponse } from './user-urls-response';
export interface UserResponse extends IdentityUser{
  addresses?: Array<UserAddressResponse>;
  city?: CityResponse;
  cityId?: string;
  country?: CountryResponse;
  countryId?: string;
  creationDate: string;
  images?: Array<UserImageResponse>;
  organizations?: Array<UserOrganizationResponse>;
  phoneNumbers?: Array<UserPhoneNumbersResponse>;
  urls?: Array<UserUrlsResponse>;
}

File: despatch.Domain/Models/Default/Response/UserResponse.cs

using Microsoft.AspNetCore.Identity;
using System;
using System.Collections.Generic;

namespace despatch.Domain.Models.Default.Response
{
    public class UserResponse : IdentityUser
    {
        public DateTime CreationDate { get; set; }

        public string CityId { get; set; }
        public string CountryId { get; set; }

        public virtual CityResponse City { get; set; }
        public virtual CountryResponse Country { get; set; }

        public ICollection<UserAddressResponse> Addresses { get; set; }
        public ICollection<UserOrganizationResponse> Organizations { get; set; }
        public ICollection<UserImageResponse> Images { get; set; }
        public ICollection<UserPhoneNumbersResponse> PhoneNumbers { get; set; }
        public ICollection<UserUrlsResponse> Urls { get; set; }

    }
}
